libsolidity/interface/StandardCompiler.cpp

```diff
@@ -316,6 +316,27 @@ Json::Value StandardCompiler::compileInternal(Json::Value const& _input)
 			return formatFatalError("JSONError", "Invalid input source specified.");
 	}
 
+	Json::Value const& auxInputs = _input["auxiliaryInput"];
+	if (!!auxInputs)
+	{
+		Json::Value const& smtlib2Responses = auxInputs["smtlib2"];
```

Comment:
Likewise we could consider naming this `smtlib2responses` or similar.

Reply:
Yes, sounds good.

Reply:
Likewise: `auxiliaryInput.smtlib2responses`.